PR: Ability to delete blogs in mutisite

esudalaimadan opened the pull request:
Fix #50, #121 

@sudar
1) please check against subdomains as I have checked only against subdirectories.
2) When running as single site, multisite tests are running and failing.

esudalaimadan committed:
Add Bulk WP menu at network level

sudar committed:
Show modules in network page

esudalaimadan committed:
Add usermeta module and test for the same.

esudalaimadan committed:
Apply fixes from StyleCI

esudalaimadan committed:
Merge pull request #674 from sudar/analysis-Xk1k7R

Apply fixes from StyleCI

esudalaimadan committed:
Add tests for registration, limit and no posts filters.

Fix #671

esudalaimadan committed:
Add is network admin condition.

esudalaimadan committed:
Make sure wpmu_delete_user is not called for requests from subsites.

esudalaimadan committed:
Fix review comment.

esudalaimadan committed:
Merge branch 'dev/6.1.0' into 671-feature/add-mutisite-support-for-users-module

sudar committed:
Merge branch 'dev/6.1.0' into 671-feature/add-mutisite-support-for-users-module

esudalaimadan committed:
Add skeleton for Bulk Delete Sites module.

esudalaimadan committed:
Delete BuddPress pending users

Fix #151

esudalaimadan committed:
Site deletion works now.

esudalaimadan committed:
Handle subdomain installations and few minor tweaks.

Fix #50

esudalaimadan committed:
Apply fixes from StyleCI

esudalaimadan committed:
Merge pull request #710 from sudar/analysis-orLMEZ

Apply fixes from StyleCI

esudalaimadan committed:
Merge branch 'dev/6.1.0' into 50-feature/delete-blogs-in-multisite

esudalaimadan committed:
Add code to handle starts with and ends with operators.

esudalaimadan committed:
Merge branch '50-feature/delete-blogs-in-multisite' of github.com:sudar/bulk-delete into 50-feature/delete-blogs-in-multisite

sudar commented:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- include/Core/Sites/SitesModule.php  4
- include/Core/Sites/Modules/DeleteSitesByNameModule.php  7
- include/Core/Users/DeleteUsersInMultisitePage.php  1
- tests/wp-unit/include/Core/Users/Modules/DeleteUsersByUserMetaInMultisiteModuleTest.php  6
- include/Core/Multisite/MultisiteAdminUIBuilder.php  3
- include/Core/Sites/DeleteSitesInMultisitePage.php  1
- include/Core/Users/UsersModule.php  2
- include/Core/Users/Modules/DeleteUsersByUserMetaInMultisiteModule.php  2
         

Clones added
============
- include/Core/Sites/SitesModule.php  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/BulkWP/bulk-delete/pullRequest?prid=4920807&bid=16284122)

sudar commented on include/Core/Sites/SitesModule.php:
```diff
@@ -0,0 +1,153 @@
+		 */
+		do_action( 'bd_before_query', $wp_site_query );
+
+		$sites = (array) $wp_comment_query->query( $options );
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused local variables such as '$wp_comment_query'.](https://app.codacy.com/manual/BulkWP/bulk-delete/pullRequest?prid=4920807)

sudar commented on include/Core/Sites/Modules/DeleteSitesByNameModule.php:
```diff
@@ -0,0 +1,107 @@
+	}
+
+	// phpcs:ignore Squiz.Commenting.FunctionComment.Missing
+	protected function build_query( $options ) {
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused parameters such as '$options'.](https://app.codacy.com/manual/BulkWP/bulk-delete/pullRequest?prid=4920807)

sudar committed:
Handle generic operator dropdown

sudar committed:
Remove contains operator

sudar committed:
Pass the correct values as arguments

esudalaimadan committed:
Replace old operator dropdowns with new generic operator logic.

Fix #706

esudalaimadan committed:
Add Unix timestamp format handling in query overrider and notice.

esudalaimadan committed:
Merge branch 'dev/6.1.0' into fix/706-generic-operators-logic

esudalaimadan committed:
Apply fixes from StyleCI

esudalaimadan committed:
Merge pull request #715 from sudar/analysis-WNxeW5

Apply fixes from StyleCI

sudar committed:
Merge pull request #714 from sudar/fix/706-generic-operators-logic

PR: generic operators logic

esudalaimadan committed:
Add help text about post deletion in Delete user modules

Fix #716

Co-authored-by: Sudar Muthu <sudar@sudarmuthu.com>

esudalaimadan committed:
Don't fetch all the fields while deleting users if it is not needed

Fix 720

Co-authored-by: Sudar Muthu <sudar@sudarmuthu.com>

esudalaimadan committed:
Add tests for deleting posts along with users

Fix #717

Co-authored-by: Sudar Muthu <sudar@sudarmuthu.com>

sudar committed:
Update NPM dependencies

sudar committed:
Update Grunt

sudar committed:
Allow short array syntax

sudar committed:
Handle legacy cron hooks in modules

sudar committed:
Change the format of the properties for PrefixAllGlobals rule

esudalaimadan committed:
Retrieve only user ids instead of user object while deleting users

Fix issues introduced by #721
Fix #720

Co-authored-by: Sudar Muthu <sudar@sudarmuthu.com>

esudalaimadan committed:
Add delete comments by id method in comments module.

Fix #729

esudalaimadan committed:
Refactor deleted count in Post module.

Fix #730

sudar committed:
Merge pull request #731 from sudar/729-fix/add-delete-comments-by-id-method

PR: Add delete comments by id method in comments module.

esudalaimadan committed:
Merge branch 'dev/6.1.0' into 730-fix/refactor-deleted-count

sudar committed:
Merge pull request #732 from sudar/730-fix/refactor-deleted-count

PR: Refactor deleted count in Post module.

dependabot[bot] committed:
Bump websocket-extensions from 0.1.3 to 0.1.4

Bumps [websocket-extensions](https://github.com/faye/websocket-extensions-node) from 0.1.3 to 0.1.4.
- [Release notes](https://github.com/faye/websocket-extensions-node/releases)
- [Changelog](https://github.com/faye/websocket-extensions-node/blob/master/CHANGELOG.md)
- [Commits](https://github.com/faye/websocket-extensions-node/compare/0.1.3...0.1.4)

Signed-off-by: dependabot[bot] <support@github.com>

sudar committed:
Merge pull request #728 from sudar/dependabot/npm_and_yarn/websocket-extensions-0.1.4

Bump websocket-extensions from 0.1.3 to 0.1.4

sudar committed:
Add LICENSE

sudar committed:
Add code of conduct file

sudar committed:
Remove deprecated add-ons

esudalaimadan committed:
Remove unnecessary td tags.

Fix #737

sudar committed:
Fix StyleCi configuration

Recently the way StyleCi parses configuration has been changed which was generating errors in config files that were working earlier. This commit fixes those errors.

sudar committed:
Merge branch 'dev/6.1.0' into 737-fix/remove-unwanted-td

sudar committed:
Merge pull request #738 from sudar/737-fix/remove-unwanted-td

PR: Remove unnecessary td tags

esudalaimadan committed:
Add crom settings to term modules.

Fix #739

esudalaimadan committed:
Merge branch 'dev/6.1.0' into 739-feature/add-cron-settings-to-term-modules

dependabot[bot] committed:
Bump lodash from 4.17.15 to 4.17.19 (#736)

Bumps [lodash](https://github.com/lodash/lodash) from 4.17.15 to 4.17.19.
- [Release notes](https://github.com/lodash/lodash/releases)
- [Commits](https://github.com/lodash/lodash/compare/4.17.15...4.17.19)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Sudar Muthu <sudar@sudarmuthu.com>

sudar committed:
Use the correct add-on code in url

sudar committed:
Add change log to history

sudar committed:
Merge branch 'dev/6.1.0' into 739-feature/add-cron-settings-to-term-modules

sudar committed:
Merge pull request #740 from sudar/739-feature/add-cron-settings-to-term-modules

PR: Add cron settings to term modules.

sudar committed:
Bump up version to 6.1.0

sudar committed:
Update composer dependencies

esudalaimadan committed:
Add the ability to use different labels for submit button and action

esudalaimadan committed:
Fix User ID retrieval from email and exclude parameter check.

Fix #744

sudar committed:
Merge pull request #746 from sudar/744-fix/user-id-retrieval-with-email

PR: Fix User ID retrieval from email and exclude parameter check.

esudalaimadan committed:
Remove quotes from translation function

Fix #747

esudalaimadan committed:
Merge branch 'dev/6.1.0' into 50-feature/delete-blogs-in-multisite

esudalaimadan committed:
Fix typos and tests.
